M2/Macaulay2/packages/FastLinAlg.m2

```diff
@@ -710,10 +760,14 @@ Rn(ZZ, Ring) := opts -> (n1, R1) -> (
     possibleMinors := binomial(n, fullRank)*binomial(numberRelations, fullRank);
     minNumberToCutDown := n1+1;
     local numberOfMinorsCompute;
-    if (opts.MaxMinors === null) then (
-        numberOfMinorsCompute = 10*minNumberToCutDown + 8*ceiling(log_1.3(possibleMinors));)
+    if (instance (opts.MaxMinors, Function)) then (
+        numberOfMinorsCompute = opts.MaxMinors(minNumberToCutDown, possibleMinors); )
+        --5*minNumberToCutDown + 8*ceiling(log_1.3(possibleMinors));)
+    else if (instance(opts.MaxMinors, Number)) then (
+        numberOfMinorsCompute = opts.MaxMinors;)
     else (
-        numberOfMinorsCompute = opts.MaxMinors;);
+        numberOfMinorsCompute = 5*minNumberToCutDown + 8*ceiling(log_1.3(possibleMinors));
```

Comment:
At this point you should signal an error if it isn't `null`.  (Users can make mistakes.)

Reply:
I think you are misreading this.  The user can pass a function, a number, or anything else.  If they pass anything else, it reverts to a default function.